modules/ROOT/pages/verify-nfs.adoc

```diff
@@ -0,0 +1,187 @@
+
+- `nc -zvu <host_name> 111`
+- `nc -zv <host_name> 2049`
+
+When the responses are successful you receive a response similar to the following:
+
+----
+[ec2-user@ip-10-1-1-97 ~]$ nc -zv <nfs_host_name> 2049
+Ncat: Version 7.50 ( https://nmap.org/ncat )
+Ncat: Connected to 10.1.0.155:2049.
+Ncat: 0 bytes sent, 0 bytes received in 0.01 seconds.
+----
+
+== Mount the NFS Server
+
+Mount the NFS server in a temporary directory on the host. 
+
+. Switch to root by typing: +
+`sudo su`
+. To mount the NFS in a temporary directory, run the following commands and replace both the `<nfs-server>` and the `<nfs-path>` with your own values. +
+Run: +
+`mkdir -p /mnt/home` +
+and then: +
+`mount -t nfs4 -o proto=tcp,port=2049 <nfs-server>:<path> /mnt/home/` +
+If the mount fails, you do not have the required permissions. 
+
+== Validate NFS Performance
+
+Perform careful analysis of your environment, both from the client and server point of view for optimal NFS performance.
+
+These are the high level steps for validating NFS performance:
+
+. Perform some basic operations to test the performance and latency of your NFS server. Specifically, these operations will: +
+* Transfer data from the client (Anypoint Platform environment) to the NFS server 
+* Measure how long each transfer takes 
+. To validate how the Anypoint Platform handles NFS files transfers, `read` and `write` operations are tested, with both large (128MB) and small (80KB) files.
+. Use the command `time dd if=/dev/zero of=/mnt/home/<nameOfFile> bs=<blockSize> count=<amountOfBlocks>` to test performance of the NFS server while transferring files with: +
+* Names `<nameOfFile>` 
+* The block size and number of blocks in the transfer are `<blockSize>` and `<amountOfBlocks>`, respectively.
+
+=== Test Writing and Reading Large files on the NFS server
+
+These `write` and `read` tests are performed using large files:
+
+* File size: 128 MB
+* Block size: 4 KB
+* Number of blocks: 32768  
+
+During the time the tests are running, different files are generated (one file per test).
+
+. To start with the testing, create a directory in which to mount the NFS server: 
++
+----
+sudo su
+mkdir -p /mnt/home
+----
+. Mount the NFS server: 
++ 
+----
+mount -t nfs4 -o proto=tcp,port=2049 <nfs-server>:<path> /mnt/home/
+----
+. Perform writes (five files of 128 MB each) on the NFS server by running the following command: 
++
+----
+time for i in {1..5}; do dd if=/dev/zero of=/mnt/home/greatfile$i.test bs=4k count=32768; done
+----
++
+The command output is similar to the following:
++
+----
+[root@ip-10-1-1-97 ec2-user]# time for i in {1..5}; do dd if=/dev/zero of=/mnt/home/greatfile$i.test bs=4k count=32768; done
+32768+0 records in
+32768+0 records out
+134217728 bytes (134 MB) copied, 2.12805 s, 63.1 MB/s
+.
+.
+.
+real 0m8.378s
+user 0m0.034s
+sys 0m0.792s
+----
++
+The `real` parameter in the test output should be less than 15 seconds. 
```

Comment:
Do we mean set the value of `real` to less than 15?
Or do we mean that the output contains a parameter `real` whose value should be smaller than 15?

Reply:
They are not setting this. It is output.